drivers/rtc/Kconfig.stm32_rtc

```diff
@@ -0,0 +1,29 @@
+# Kconfig - STM32 RTC configuration options
+#
+# Copyright (c) 2018 Workaround GmbH
+# Copyright (c) 2018 Allterco Robotics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config RTC_STM32
+	bool "STM32 RTC Driver"
+	depends on SOC_FAMILY_STM32 && SOC_SERIES_STM32L4X
+	default n
+	select USE_STM32_LL_RTC
+	select USE_STM32_LL_PWR
+	select USE_STM32_LL_RCC
+	select USE_STM32_LL_EXTI
+	help
+	  Build STM32 RTC driver.
+
+if RTC_STM32
+
+config RTC_ALARM_SECOND
```

Comment:
Could you use rtc node `prescaler` property from DT instead of adding a Kconfig option?

Reply:
Added, along with a line in `arm/soc/st_stm32/stm32l4/dts.fixup`

    #define CONFIG_RTC_0_PRESCALER                  ST_STM32_RTC_40002800_PRESCALER

However, `rtc.h` expects `CONFIG_RTC_PRESCALER`, not sure where this mapping should go

Reply:
If there is only one RTC then you can just use CONFIG_RTC_*, like we did for USB interface.